torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1614,7 +1614,7 @@ def sample_inputs_new_fns(self, device, dtype, requires_grad, *, is_strided=Fals
 def sample_inputs_empty_strided(op, device, dtype, requires_grad=False, **kwargs):
 
     inputs = [
-        ((), (), {}),
+        ((), (), {'dtype': dtype, 'device': device}),
         ((S,), (4,), {'dtype': dtype, 'device': device}),
```

Comment:
@ngimel, `test_multiple_devices` failed because, in this case, it created `cpu` tensors by default, and the output is expected to be in the `cuda` device for this test. Thanks, @kshitij12345, for finding this!

This test was always skipped because my system does not support testing on multiple devices.